### DISCUSSION TITLE
Alpha changes

### DISCUSSION THREAD

**Discussion by mmb25** [fuatakgun/eufy_security#627] [General]
Hello guys, I can see that a lot of work has been done on the alpha and will be merged into master in a week.

I would like to know what is the scope of this new version, is it focused on bug fixing, or does it have some new features ?

I am specially interested in be able to retrieve the name of the detected person on a homebase 3 environment (It exist the sensor, but apparently never gets populated)

Thank you for your great work!

**Comment by fuatakgun** [comment]
1. Code is becoming manageable in the long term (technically: i am fixing my architectural mistakes). It would be easy for any external dev to step in (maintainability) I don't want to maintain current version anymore.
2. Person name sensor exists on both master and alpha versions and if they are not working, please reach out to bropat over eufy-security-ws. If the information is not flowing from his repository, i am blocked.
